next.config.js

```diff
@@ -0,0 +1,9 @@
+/* 
+    OpenSSL 3 does not support md4 any more, but webpack hardcodes it all over the place in this way.
```

Comment:
```suggestion
    OpenSSL 3 does not support md4 any from node version 16, but webpack hardcodes it all over the place in this way.
```

I'm pretty sure that this code will be available also with deno, so we need to specify the env.

Reply:
Done